src/dynamodb.ts

```diff
@@ -252,20 +319,19 @@ function dynamoDBTestHooks (useUniqueTables = false, opts) {
   };
 }
 
-exports.getConnection = getConnection;
-exports.createTables = createTables;
-exports.destroyTables = destroyTables;
-exports.dynamoDBTestHooks = dynamoDBTestHooks;
-exports.launchDynamoContainer = launchDynamoContainer;
-
 /**
- * @param {TestInterface} test ava object
+ * @param {TestInterface} anyTest ava object
  * @param {boolean} useUniqueTables
- * @param {object} opts
+ * @param {DynamoDBTestOptions} opts
  * @param {boolean} opts.inMemory Determines whether to run the local DynamoDB instance
  * in in-memory mode, or to persist the data to disk
  */
-exports.useDynamoDB = (test, useUniqueTables, opts) => {
+export function useDynamoDB (
+  anyTest: TestInterface,
+  useUniqueTables?: boolean,
+  opts?: DynamoDBTestOptions
+): void {
+  const test = anyTest as TestInterface<DynamoDBTestContext<any>>
```

Comment:
If anyTest must be a `TestInterface<DynamoDBTestContext<any>>` could it be moved up to the `anyTest` argument to catch more errors?

Reply:
With the way `ava` works, `TestInterface` doesn't actually have anything set.  The type only matters for the test context inside each `test('name', t: Implementation<Context> => {})`.  So it could be set at the top level, forcing lib users to set the context before, or internally and work just the same.